diesel/src/type_impls/mod.rs

```diff
@@ -0,0 +1,7 @@
+mod date_and_time;
+pub mod floats;
```

Comment:
floats and option should be private, not public

Reply:
This code was just moved unchanged.

Reply:
Git does not see this as a code-move (it is shown in the diff). Therefore we could make those two also modules private (they do not contain any public items, as far as I have seen)

Reply:
Yeah, the replacement of `types` with `sql_types` probably touched enough lines for it to not be seen as a rename in git.